File: javascript 90 questions/Q29.js


let arr=[11,15,0,9,8,1];
let target=9;
let length=arr.length;
for(i=0;i<length;i++){
    for(j=i+1;j<length;j++){
        if(arr[i]+arr[j]==target){
            console.log(i,j);
}
}
}

File: C questions/Q29.c


#include <stdio.h>
int main(){
    int arr[]={2,7,11,15};
    int target=9;
    int length=sizeof(arr)/sizeof(arr[0]);
    for(int i=0;i<length;i++){
        for(int j=i+1;j<length;j++){
            if(arr[i]+arr[j]==target){
                printf("%d\n",i);
                printf("%d",j);
            }
 }
}
}